src/anomalib/utils/metrics/thresholding/base.py

```diff
@@ -0,0 +1,33 @@
+"""Base class for thresholding metrics."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+from abc import ABC
+from typing import Any
+
+from torchmetrics import Metric
+
+
+class BaseAnomalyScoreThreshold(Metric, ABC):
+    """Base class for thresholding metrics."""
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+
+    def compute(self) -> Any:
+        """Compute the threshold
+        Returns:
```

Comment:
```suggestion
        
        Returns:
```

Reply:
I thought we had a pre-commit hook to check docstrings.

Reply:
yeah, weird. It should've picked it up